docs/Building/android.md

```diff
@@ -79,9 +79,7 @@ We have a gradle task setup for that.
 
 If you need to work with subscriptions or other play store functionality in the emulator, you will need to sign the apk.
 
-Make sure have an environment variable `AUTOGRAPH_TOKEN` set (ask a fellow dev).
+There formerly was a `sign.sh` script that helped with this. The server it relied no longer exists.
+TaskCluster builds should be signed, and so creating a draft PR should create a signed build that could be used.
```

Comment:
Creating a draft pr does NOT create a real signed build. 
Otherwise any of us could just create a WIP PR with evilthings and publish that as an official mozilla vpn apk. 

Draft pr's are signed same as local builds using this key but that is just convinence
(as otherwise you'd need to uninstall the app when switchting from tc-pr builds to local builds) : 
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/android/debug.keystore